src/lib/objects.js

```diff
@@ -0,0 +1,74 @@
+const createPerson = (name, age) => {
+  const person = {
+    name,
+    age
+  };
+  return person;
+};
+
+const getName = object => {
+  return object.name;
+};
+
+const getProperty = (property, object) => {
+  return object[property];
+};
+
+const hasProperty = (property, object) => {
+  return object.hasOwnProperty(property);
+};
+
+const isOver65 = person => {
+  return person.age > 65;
+};
+
+const getAges = people => {
+  const ages = [];
+  for (let i = 0; i < people.length; i++) {
+    ages.push(people[i].age);
+  }
+  return ages;
```

Comment:
This could be re-written. You can make use of built in methods like filter and map to the same job for you  a bit more concisely:
```
  return people.filter(person => person.name).map(person => person.age);
```